### ISSUE TITLE
Content remove when insert path in source code editor

### ISSUE THREAD

**Issue by rsenet** [ckeditor/ckeditor4#4154]
Hello,

Sorry for open a new issue but I think the answer was not complete. As specified in https://github.com/ckeditor/ckeditor4/issues/4138, the content of the textarea is inserted in a MySQL database. The content is (withtout additional spaces):

&lt ; Directory /usr/local/apache2/htdocs/listme & gt ;
  Options +Indexes
&lt ; /Directory & gt;

During the insertion, it is working. However, When I want to edit the data, I get them from the database and here, all the content is removed.

In summary, I can add content without any problem but I cannot update it. I tried to disable ACF (allowedContent: true) withtout any success.

Any additionnal idea?

_Originally posted by @rsenet in https://github.com/ckeditor/ckeditor4/issues/4138#issuecomment-654287528_

**Comment by Dumluregn** [comment]
Hello, I'm unable to reproduce your problem - as you can see in [my CodePen example](https://codepen.io/dumluregn/pen/QWyxEMW?editors=1010), if `allowedContent = true` option is active, editor content is not modified neither at initialisation nor when mode is toggled. Please provide us a demo where your content is removed.

**Comment by rsenet** [comment]
> Hello, I'm unable to reproduce your problem - as you can see in [my CodePen example](https://codepen.io/dumluregn/pen/QWyxEMW?editors=1010), if `allowedContent = true` option is active, editor content is not modified neither at initialisation nor when mode is toggled. Please provide us a demo where your content is removed.

Hello Dariusz, thank you for your answer! I just sent you a private mail (address specified in your profil) with a demonstration and the source code. :)

**Comment by Dumluregn** [comment]
Hi!

I analysed the example you sent me (however please note that it would be easier to follow - also for others - if the whole communication were in the same channel). I inserted the following vulnerability to your dashboard (named `New2`):
```
&lt;Directory /usr/local/apache2/htdocs/listme&gt;
Options + Indexes
&lt;/Directory&gt;
```

And as you can see during the upload or download the [HTML entities](https://dev.w3.org/html5/html-author/charref) I used were transformed to characters and as a result the data that goes to the editor is:

```
<directory apache2="" htdocs="" listme="" local="" usr=""> Options + Indexes </directory>
```

So you probably should analyse your PHP code and modify it to prevent HTML entities conversion, because as you can see in [my example](https://codepen.io/dumluregn/pen/QWyxEMW?editors=1010), CKEditor doesn't do it on its own and if the content stayed the way it was inserted, it would be displayed correctly.

After all please remember, that our GitHub issue tracker serves for reporting bugs and new features only. A perfect place for requests like yours is [Stack Overflow](https://stackoverflow.com/questions/tagged/ckeditor) where our team and the community assist other users in solving their issues. If you have CKEditor license, please contact our support team for further implementation help.
